src/Run/CommandLine.cs

```diff
@@ -0,0 +1,1876 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.DotNet.Execute;
+
+using System.Reflection;
+
+namespace Microsoft.Fx.CommandLine
```

Comment:
Any ideas of how much of this code you are actually using? I don't like the fact that it is forked and would prefer that we make changes to the core version of this file in a compatible way, but if that isn't realistic then I would prefer we prune any dead code in your fork given there is no plans to try and merge it back.